src/View/StringTemplate.php

```diff
@@ -309,11 +310,11 @@ public function formatAttributes(?array $options, ?array $exclude = null): strin
      * Works with minimized attributes that have the same value as their name such as 'disabled' and 'checked'
      *
      * @param string $key The name of the attribute to create
-     * @param array<string>|string $value The value of the attribute to create.
+     * @param array<string>|string|float|int|bool $value The value of the attribute to create.
      * @param bool $escape Define if the value must be escaped
      * @return string The composed attribute.
      */
-    protected function _formatAttribute(string $key, $value, $escape = true): string
+    protected function _formatAttribute(string $key, array|string|float|int|bool $value, bool $escape = true): string
```

Comment:
Added all the types that were passed in.

Reply:
There's a failing test since it receives a `FrozenTime` instance too as value. I think the type should be just set to `mixed` since it can accept any scalar or stringable value.